components/server/src/websocket/websocket-connection-manager.ts

```diff
@@ -237,11 +238,15 @@ export class WebsocketConnectionManager implements ConnectionHandler {
         }
 
         const clientHeaderFields: ClientHeaderFields = {
-            ip: takeFirst(expressReq.ips),
+            ip: clientIp(expressReq),
             userAgent: expressReq.headers["user-agent"],
             dnt: takeFirst(expressReq.headers.dnt),
             clientRegion: takeFirst(expressReq.headers["x-glb-client-region"]),
         };
+        // TODO(gpl): remove once we validated the current approach works
+        log.debug("masked wss client IP", {
+            maskedClientIp: maskIp(clientHeaderFields.ip),
+        });
```

Comment:
Do you want to keep this in the PR or wanted to remove it before merging?

Reply:
I wanted this to hit prod so we can turn it on if we want; and remove it in a subsequent PR. :+1: